docs/software/file_servers.md

```diff
@@ -163,18 +163,47 @@ Network file system server.
     - Address = IP address of your DietPi system (e.g.: 192.168.0.100)
     - Port = 2049
 
-=== "Increase security"
+=== "Access configuration"
 
-    By default NFS will allow anyone to connect. You can limit who can access the NFS share by setting a IP address range:
+    The NFS access configuration is done via **export files**.  
+    You can edit the `/etc/exports` file as well as adding further export files within the `/etc/exports.d` directory.
 
-    - Edit the following file: `/etc/exports`
+    **Explanations of the export file format** are available in the internet or can be read in the man pages (use `man exports`, therefore the package **man** needs to be installed).
+
+    After changing the access configuration, the export information can be re-read via command
+
+    ```
+    /usr/sbin/exportfs -ra
```

Comment:
Let's remove the path, as it's in default PATH => `exportfs -ra`

Reply:
if /usr/sbin is within your PATH, then it's o.k.
This is not always the case in DietPi environments.

But nevertheless, I will remove it.

Reply:
It matches also the calls below.

/bin /sbin /usr/bin /usr/sbin /usr/local/bin and /usr/local/sbin must all be part of the PATH and it's default in /etc/profile and hardcoded default in bash, if it's not set on start. In our scripts we also set this PATH forcefully. Non-root logins can miss the sbin ones, but in that case it is also not intended that the contained commands are called 😄.

Another topic that would make a good knowledge-base section in the docs 🙂.